app/models/registration.rb

```diff
@@ -521,4 +521,56 @@ def competition_events_changed?
   def serializable_hash(options = nil)
     super(DEFAULT_SERIALIZE_OPTIONS.merge(options || {}))
   end
+
+  def attempt_auto_accept
+    return false if Rails.env.production? && EnvConfig.WCA_LIVE_SITE?
```

Comment:
This should be moved into a `Registration.AUTO_ACCEPT_ENABLED` setting, similar to how we already have `PaypalInterface.paypal_disabled?`

Reply:
Added `auto_accept_enabled?` method to contain this